### PR TITLE
Docs add windows versions of commands

### PR DESCRIPTION
**Pull request by mpuckett159**
This addresses #12968 but I don't think it completely resolves/closes it so I'll leave that up to the issue creator/maintainers to decide.

This PR adds windows versions of commands where I know them/as they exist, and implements the tabbing feature for these cases.

Also updated an existing case that was using the card style for multi-OS command examples.

### CODE REVIEW

File: site/content/en/docs/handbook/config.md [hunk 1]
```diff
@@ -124,14 +124,42 @@ Some features can only be accessed by minikube specific environment variables, h
...
+{{% /windowstab %}}
+{{% /tabs %}}
+
 ### Making environment values persistent
 
 To make the exported variables persistent across reboots:
 
 * Linux and macOS: Add these declarations to `~/.bashrc` or wherever your shells environment variables are stored.
-* Windows: Add these declarations via [system settings](https://support.microsoft.com/en-au/help/310519/how-to-manage-environment-variables-in-windows-xp) or using [setx](https://stackoverflow.com/questions/5898131/set-a-persistent-environment-variable-from-cmd-exe)
+* Windows: Either add these declarations to your `~\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1` or run the following in a powershell terminal:
```
Comment (spowelljr) [inline site/content/en/docs/handbook/config.md:162]:
```suggestion
* Windows: Either add these declarations to your `~\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1` or run the following in a PowerShell terminal:
```

File: site/content/en/docs/handbook/pushing.md [hunk 2]
```diff
@@ -54,9 +54,29 @@ This means you don't have to build on your host machine and push the image into
...
 eval $(minikube docker-env)
 ```
+{{% /linuxtab %}}
+{{% mactab %}}
+```shell
+eval $(minikube docker-env)
+```
+{{% /mactab %}}
+{{% windowstab %}}
+Powershell
```
Comment (spowelljr) [inline site/content/en/docs/handbook/pushing.md:69]:
```suggestion
PowerShell
```

File: site/content/en/docs/handbook/pushing.md [hunk 3]
```diff
@@ -184,8 +211,22 @@ Note: On macOS the remote client is called "podman", since there is no local "po
 
 {{% /mactab %}}
 {{% windowstab %}}
+This is similar to docker-env but only for CRI-O runtime.
+To push directly to CRI-O, configure podman client on your host using the podman-env command in your shell:
+
+Powershell
```
Comment (spowelljr) [inline site/content/en/docs/handbook/pushing.md:217]:
```suggestion
PowerShell
```

File: site/content/en/docs/handbook/pushing.md [hunk 4]
```diff
@@ -184,8 +211,22 @@ Note: On macOS the remote client is called "podman", since there is no local "po
...
+
+cmd
+```shell
+@FOR /f "tokens=*" %i IN ('minikube -p minikube podman-env --shell cmd') DO @%i
+```
 
-now you can 'build' against the storage inside minikube, which is instantly accessible to kubernetes cluster.
+You should now be able to use podman client on the command line on your host machine talking to the podman service inside the minikube VM:
+
+Now you can 'build' against the storage inside minikube, which is instantly accessible to kubernetes cluster.
```
Comment (spowelljr) [inline site/content/en/docs/handbook/pushing.md:229]:
```suggestion
Now you can 'build' against the storage inside minikube, which is instantly accessible to Kubernetes cluster.
```

File: site/content/en/docs/handbook/pushing.md [hunk 5]
```diff
@@ -167,12 +186,20 @@ Note: On Linux the remote client is called "podman-remote", while the local prog
 
 {{% /linuxtab %}}
 {{% mactab %}}
+This is similar to docker-env but only for CRI-O runtime.
+To push directly to CRI-O, configure podman client on your host using the podman-env command in your shell:
```
Comment (spowelljr) [inline site/content/en/docs/handbook/pushing.md:190]:
```suggestion
To push directly to CRI-O, configure Podman client on your host using the podman-env command in your shell:
```

File: site/content/en/docs/handbook/pushing.md [hunk 6]
```diff
@@ -167,12 +186,20 @@ Note: On Linux the remote client is called "podman-remote", while the local prog
...
+
+```shell
+eval $(minikube podman-env)
+```
+
+You should now be able to use podman client on the command line on your host machine talking to the podman service inside the minikube VM:
```
Comment (spowelljr) [inline site/content/en/docs/handbook/pushing.md:196]:
```suggestion
You should now be able to use Podman client on the command line on your host machine talking to the Podman service inside the minikube VM:
```

File: site/content/en/docs/handbook/pushing.md [hunk 7]
```diff
@@ -184,8 +211,22 @@ Note: On macOS the remote client is called "podman", since there is no local "po
 
 {{% /mactab %}}
 {{% windowstab %}}
+This is similar to docker-env but only for CRI-O runtime.
+To push directly to CRI-O, configure podman client on your host using the podman-env command in your shell:
```
Comment (spowelljr) [inline site/content/en/docs/handbook/pushing.md:215]:
```suggestion
To push directly to CRI-O, configure Podman client on your host using the podman-env command in your shell:
```

File: site/content/en/docs/handbook/pushing.md [hunk 8]
```diff
@@ -184,8 +211,22 @@ Note: On macOS the remote client is called "podman", since there is no local "po
...
+& minikube -p minikube podman-env --shell powershell | Invoke-Expression
+```
+
+cmd
+```shell
+@FOR /f "tokens=*" %i IN ('minikube -p minikube podman-env --shell cmd') DO @%i
+```
 
-now you can 'build' against the storage inside minikube, which is instantly accessible to kubernetes cluster.
+You should now be able to use podman client on the command line on your host machine talking to the podman service inside the minikube VM:
```
Comment (spowelljr) [inline site/content/en/docs/handbook/pushing.md:227]:
```suggestion
You should now be able to use Podman client on the command line on your host machine talking to the Podman service inside the minikube VM:
```

File: site/content/en/docs/handbook/pushing.md [hunk 9]
```diff
@@ -167,12 +186,20 @@ Note: On Linux the remote client is called "podman-remote", while the local prog
...
+```
+
+You should now be able to use podman client on the command line on your host machine talking to the podman service inside the minikube VM:
 
 ```shell
 podman help
 ```
 
-now you can 'build' against the storage inside minikube, which is instantly accessible to kubernetes cluster.
+Now you can 'build' against the storage inside minikube, which is instantly accessible to kubernetes cluster.
```
Comment (spowelljr) [inline site/content/en/docs/handbook/pushing.md:202]:
```suggestion
Now you can 'build' against the storage inside minikube, which is instantly accessible to Kubernetes cluster.
```

**Review by spowelljr — CHANGES_REQUESTED** [review]
Our documentation hasn't respected capitalization of proper nouns, so just trying to improve that area with new changes.

File: site/content/en/docs/handbook/config.md (hunk 1 above)
Comment (mpuckett159) [reply site/content/en/docs/handbook/config.md:162]: TiL that it's actually PowerShell and not Powershell lol.

**Review by spowelljr — APPROVED** [review]
Thanks for the contribution! 👍

**Pull request merged**